src/main/java/seedu/resireg/ui/PiechartsTab.java

```diff
@@ -0,0 +1,8 @@
+package seedu.resireg.ui;
+
+/**
+ * Represents a tab containing a {@code piechartPanel}. The tab may contain other UI elements.
+ */
+interface PiechartsTab extends TabContainer {
+    void setPiechartPanel(PiechartPanel piechartPanel);
```

Comment:
What's the reason for this interface?

Reply:
The interface is for future-proofing if we ever want to have a list of piecharts.